### ISSUE TITLE
Fps issues

### ISSUE THREAD

**Issue by Kirikou12312** [notnotnotswipez/BonelabMultiplayerMockup#7]

**Comment by Kirikou12312** [comment]
So me and my friend tried to launch Mockup, everything was right at first but then, we see each others lagging and having bad fps too.  I see my friend having very low fps. Any advice?

**Comment by notnotnotswipez** [comment]
Play Fusion, Mockup is an unfinished unpolished experience.

**Comment by Kirikou12312** [comment]
Yes but i have a problem with fusion, i don't know how to put beta branch on oculus app. ( i don't have bonelab on steam )
